sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/elasticsearch/SmallITCluster/elasticsearch-svc-rc.yaml

```diff
@@ -47,6 +47,18 @@ spec:
     metadata:
```

Comment:
I realized this is still set up as a public LoadBalancer kubernetes service - we should switch it NodePort and add the separate elasticsearch-service-for-local-dev.yaml  that exposes a public LoadBalancer service. 

Same for the large elasticsearch cluster if that hasn't been similarly switched.